lib/vrl/tests/tests/issues/13461_assignment_panics.vrl

```diff
@@ -0,0 +1,8 @@
+# result: true
+
+.@foo = true
```

Comment:
Each line should be a separate test here. The panic is an underflow of the source code, so only the first one will actually panic.

Reply:
Good catch 👍 working on an update to this PR.